codemp/botlib/be_interface.cpp

```diff
@@ -910,5 +910,7 @@ botlib_export_t *GetBotLibAPI(int apiVersion, botlib_import_t *import) {
 	be_botlib_export.BotLibUpdateEntity = Export_BotLibUpdateEntity;
 	be_botlib_export.Test = BotExportTest;
 
+	PC_Init();
```

Comment:
This is a little bit ugly to call it here, but necessary. Otherwise we'd have to drill it out further through other exported APIs and just call it from there :shrug:
Let's just assume if you're getting any botlib exports, you want it to be ready for use.